src/Tests.Microsoft.Security.Utilities.Core/IdentifiableScanTests.cs

```diff
@@ -66,10 +65,16 @@ public void IdentifiableScan_IdentifiableKeys()
                     {
                         foreach (string signature in identifiable.Signatures!)
                         {
-                            string key = IdentifiableSecrets.GenerateBase64KeyHelper(seed,
-                                                                                     identifiable.KeyLength,
-                                                                                     signature,
-                                                                                     identifiable.EncodeForUrl);
+                            // Special case: Azure Search identifiable keys must be base62 encoded.
+                            bool isBase62 = signature == IdentifiableMetadata.AzureSearchSignature;
+                            string key;
+                            do
+                            {
+                                key = IdentifiableSecrets.GenerateBase64KeyHelper(seed,
+                                                                                  identifiable.KeyLength,
+                                                                                  signature,
+                                                                                  identifiable.EncodeForUrl);
+                            } while (isBase62 && key.IndexOfAny(['-', '_', '/', '+']) >= 0);
```

Comment:
>IndexOfAny [](http://example.com/codeflow?start=53&length=10)

if you wanted to, you could restrict these characters to either `+` and `/`  *or* `-` and `_` conditionally based on `identifiable.EcnodeForUrl`, as the generate base64 key helper will only propose characters from one of these two special character sets.

The reason to do so would mostly be for code clarity, right? likely nothing significant from perf with IndexOfAny, given what you've said about its underlying impl